utils/node_local_test.py

```diff
@@ -3102,8 +3136,7 @@ def _post(self, rc):
             self.aft.wf.add(self.fi_loc,
                             'NORMAL',
                             "Incorrect stderr '{}'".format(stderr),
-                            mtype='Out of memory not reported '
-                            'correctly via stderr')
+                            mtype='Out of memory not reported incorrectly via stderr')
```

Comment:
Double negative: Incorrectly or correctly?
```suggestion
                            mtype='Out of memory not reported correctly via stderr')
```

Reply:
Good catch.  A good reason not to have line-wrapped strings or double negatives in the code.